# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../linux-header/PATCHv2-4-4-scsi-export-scsi_tgt_if.h.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From patchwork Tue Sep 29 10:34:45 2009
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [PATCHv2,4/4] scsi: export scsi_tgt_if.h
Date: Tue, 29 Sep 2009 10:34:45 -0000
From: Michael S. Tsirkin <mst@redhat.com>
X-Patchwork-Id: 50526

The header scsi_tgt_if.h is used by the SCSI target
framework (tgt) (http://stgt.berlios.de/).
Export it from kernel.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>

---
include/scsi/Kbuild |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/scsi/Kbuild b/include/scsi/Kbuild
index a61932c..d798db8 100644
--- a/include/scsi/Kbuild
+++ b/include/scsi/Kbuild
@@ -4,3 +4,4 @@ header-y += scsi.h
 header-y += scsi_netlink.h
 header-y += scsi_netlink_fc.h
 header-y += scsi_bsg_fc.h
+header-y += scsi_tgt_if.h
